web/models/questionnaire.ex

```diff
@@ -16,6 +16,7 @@ defmodule Ask.Questionnaire do
     field :default_language, :string
     field :valid, :boolean
     field :deleted, :boolean
+    field :partial_config, JSON
```

Comment:
Maybe call it

```suggestion
    field :partial_relevant_config, JSON
```

🙈  ?

Reply:
✅done